### PR TITLE
feat: Add workflow dispatch to prisma/prisma-fmt-wasm

### PR DESCRIPTION
**Pull request by janpio**
Includes both version string of just released packages but also the original engine commit hash as input.

### CODE REVIEW

**Review by tomhoule — APPROVED** [review]
:+1:

**Comment by janpio** [comment]
Waiting with merge until the equivalent workflow is implemented on the prisma-fmt-wasm side.

**Comment by Jolg42** [comment]
The repo is https://github.com/prisma/prisma-fmt-wasm

**Comment by janpio** [comment]
As we are publishing to different dist tags here, we also need to do the same over in the wasm repo: https://www.npmjs.com/package/@prisma/engines
image
I'll add that here as an additional input.

File: .github/workflows/publish-engines.yml
Comment (janpio) [inline .github/workflows/publish-engines.yml:42]: `new_prisma_version` was only set in publish.ts if `npmDistTag == latest`.
This is now changed to _always_ be set, but the Npm dist tag itself is also made available as a variable.
Use that new variable instead to achieve the same result.

File: .github/workflows/publish-engines.yml
Comment (janpio) [inline .github/workflows/publish-engines.yml:56]: new dispatch to `prisma-fmt-wasm` repo on the `new-engines-version` workflow, with 3 input values: `engineHash`, `enginesWrapperVersion`, and `npmDistTag`.

File: scripts/publish.ts
Comment (janpio) [inline scripts/publish.ts:62]: always set variables instead of only on `npmDistTag == latest` so they are always available to use and make decisions.

**Comment by janpio** [comment]
This is now updated and ready for review again.

Goal of the changes: Only add an additional workflow dispatch in the other repo with 3 values as input. Existing behavior should stay unchanged.

File: scripts/publish.ts
Comment (Jolg42) [inline scripts/publish.ts:58]:
```suggestion
```

It's a duplicate of the above

**Review by Jolg42 — APPROVED** [review]
Nice! Looks good!

**Comment by janpio** [comment]
Thanks for the review. We will wait until after the release tomorrow to merge, to not potentially create additional work to fix the pipeline here.

**Review by millsp — APPROVED** [review]

**Pull request merged**
